Squablr/Views/CardView.m

```diff
@@ -30,15 +30,31 @@ - (instancetype)initWithBounds:(CGRect)frame profile:(Profile *)profile {
     if (self) {
         [self setup];
         self.profile = profile;
+        self.profilePictureIndex = 0;
+        // Creating invisible half views for the tap gesture recognizer
+        UIView *leftView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 175, 500)];
```

Comment:
Not super important but generally when we're setting frames / constraints like this we either want to use auto layout or do some sort of "manual" auto layout by figuring out what this frame should be based on the frames of the relevant views

Reply:
I thought when I use initWithFrame it specifies where on the screen the view is? Should it be paired with auto layout regardless?